client/src/Register/Register.js

```diff
@@ -20,6 +24,34 @@ export default function Register() {
     const [usernameValid, setUsernameValid] = useState(false);
     const [passwordValid, setPasswordValid] = useState(false);
 
+    const handleSubmit = async (e) => {
+        const form = e.currentTarget;
+        e.preventDefault();
+
+        const requestOptions = {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify({
+                Username: username,
+                Email: email,
+                Password: password
+            })
+        };
+
+        fetch('https://localhost:5001/api/auth/register', requestOptions)
```

Comment:
look into configs and string interpolation for domain changes

Reply:
Done